src/import_collector.rs

```diff
@@ -60,7 +60,7 @@ impl ImportCollector {
             return;
         }
         let Some(path_segment) = path.segments.first() else { return };
-        let ident = path_segment.ident.to_string();
+        let ident = path_segment.ident.unraw().to_string();
```

Comment:
We probably want a function for extracting this unraw string, just in case we find more of these usages in the future.